databases/week1/class/homework.sql

```diff
@@ -0,0 +1,83 @@
+-- Find out how many tasks are in the task table
+SELECT 
+  COUNT(title) 
+FROM 
+  task;
+  
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+  COUNT(title) 
+FROM 
+  task 
+WHERE 
+  due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  status.name LIKE "%done%";
+
+-- Find all the tasks that are not marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  NOT status.name LIKE "%done%";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT 
+  * 
+FROM 
+  task 
+ORDER BY 
+  created DESC;
+
+-- Get the single most recently created task
+SELECT 
+  MAX(created) 
+FROM 
+  task;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT 
+  title, 
+  due_date 
+FROM 
+  task 
+WHERE 
+  title LIKE '%database%';
+
+-- Get the title and status (as text) of all tasks
+SELECT 
+  task.title, 
+  status.name 
+FROM 
+  task 
+  INNER JOIN status ON task.status_id = status.id;
+
+-- Get the name of each status, along with a count of how many tasks have that status
+SELECT 
+  COUNT(task.title), 
```

Comment:
COUNT(task.title) AS COUNT
_Pointer_
For columns that you create provide a column name